frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -81,73 +79,202 @@ export function Experiment(): JSX.Element {
+                                            </Col>
+                                        </Row>
+                                        <Row className="preview-row">
+                                            <Col span={12}>
+                                                <div className="card-secondary">Target Participant Count</div>
+                                                <div className="pb">
+                                                    <span className="l4">~{sampleSize}</span> persons
+                                                </div>
+                                                <div className="card-secondary">Baseline Conversion Rate</div>
+                                                <div className="l4">{conversionRate.toFixed(1)}%</div>
+                                            </Col>
+                                            <Col span={12}>
+                                                <div className="card-secondary">Target duration</div>
+                                                <div>
+                                                    <span className="l4">~{runningTime}</span> days
+                                                </div>
+                                            </Col>
+                                        </Row>
+                                        <Row className="preview-row">
+                                            <Col>
+                                                <div className="l4">
+                                                    Conversion goal threshold
+                                                    <Tooltip
+                                                        title={`The minimum % change in conversion rate you care about. 
+                                                This means you don't care about variants whose
+                                                conversion rate is between these two percentages.`}
+                                                    >
+                                                        <InfoCircleOutlined style={{ marginLeft: 4 }} />
+                                                    </Tooltip>
+                                                </div>
+                                                <div className="pb text-small text-muted">
+                                                    Apply a threshold to broaden the acceptable range of conversion
+                                                    rates for this experiment. The acceptable range will be the baseline
+                                                    conversion goal +/- the goal threshold.
+                                                </div>
+                                                <div>
+                                                    <span className="l4 pr">Threshold value</span>
+                                                    <Slider
```

Comment:
Any particular reason you prefer the slider over the mock here?

<img width="506" alt="Screen Shot 2021-12-16 at 9 34 06 AM" src="https://user-images.githubusercontent.com/25164963/146391166-d1ba265f-1ef4-4872-ba55-1e4b585ea11b.png">

Reply:
Yep yep! More interactive, plus slider tells you the range of possible values more easily than a free form input box.

.. and it looks cool (to me) to slide and see those values change! 

What do you think?

Mentioned the same in the mocks, waiting for Chris's input as well